### PR TITLE
add overlay

### PR DESCRIPTION
**Pull request by miyabitanimchi**
I added an overlay. 
I commented the color out ("bg-gray-300" and "opacity-50") but you can uncomment them how the overlay looks like. 
Please check when you have time, thank you! 😀

### CODE REVIEW

File: client/src/common/components/Header.tsx [hunk 1]
```diff
@@ -1,87 +1,89 @@
...
+          </Link>
+        </ul>
+      </div>
+      <div
+        className={[
+          "z-10",
+          "w-full",
+          "h-screen",
+          // "bg-gray-300",
+          // "opacity-50",
```
Comment (shiv-chan) [inline client/src/common/components/Header.tsx:79]:
```suggestion
          'backdrop-filter',
	  'backdrop-blur-sm',
```
**Suggestion**: make the background blurry instead.
スクリーンショット 2021-11-29 11 10 23

File: client/src/common/components/Header.tsx [hunk 2]
```diff
@@ -1,87 +1,89 @@
...
+          "fixed",
+          isOpen ? null : "hidden",
+        ].join(" ")}
```
Comment (shiv-chan) [inline client/src/common/components/Header.tsx:82]: Smart! 
I learned a new way to set a `className` 📝  
Thanks.

**Review by shiv-chan — APPROVED** [review]
I replaced double quotations with single quotations.

We can discuss the style of the overlay later. Other than that part, looks good to me!
Thanks for your work, @miyabitanimchi 💜

**Review by Dannyboy222 — APPROVED** [review]
👍

**Pull request merged**